app/src/main/java/com/infomaniak/swisstransfer/ui/MainActivity.kt

```diff
@@ -63,12 +73,20 @@ class MainActivity : ComponentActivity(), AppReviewManageable {
     lateinit var transferManager: TransferManager
 
     override val inAppReviewManager by lazy { InAppReviewManager(this) }
+    override val inAppUpdateManager by lazy { InAppUpdateManager(this) }
 
     override fun onCreate(savedInstanceState: Bundle?) {
         enableEdgeToEdge(statusBarStyle = SystemBarStyle.dark(Color.TRANSPARENT))
         super.onCreate(savedInstanceState)
 
         initAppReviewManager()
+        initAppUpdateManager()
+
+        NetworkConfiguration.init(
+            appId = BuildConfig.APPLICATION_ID,
+            appVersionName = BuildConfig.VERSION_NAME,
+            appVersionCode = BuildConfig.VERSION_CODE,
+        )
```

Comment:
Since `MainActivity` isn't the only entry-point of the app, this could be skipped.
Submitted #527 to fix this.